contrib/forms/prior_auth/FormPriorAuth.class.php

```diff
@@ -1,6 +1,6 @@
 <?php
 
-
+use OpenEMR\ORDataObject\ORDataObject;
```

Comment:
note psr2 requires just 1 empty line after the use command (that is why travis is failing)

Reply:
should we change the file name to `.php` from `.class.php`?

Reply:
Could consider this in future PR. Note it would require also changing name where this file is required in other scripts.